app/charts/shared/interaction/tooltip-box.tsx

```diff
@@ -27,51 +74,60 @@ export const TooltipBox = ({
 }: TooltipBoxProps) => {
   const triangle = mkTriangle(placement);
   const theme = useTheme();
+
+  const [pos, posRef] = usePosition();
   return (
-    <Box
-      style={{
-        zIndex: 2,
-        position: "absolute",
-        left: x! + margins.left,
-        top: mxYOffset(y!, placement) + margins.top,
-        pointerEvents: "none",
-        transform: mkTranslation(placement),
-      }}
-    >
-      <Box
-        sx={{
-          width: "fit-content",
-          padding: 3,
-          whiteSpace: "nowrap",
-          pointerEvents: "none",
-          backgroundColor: "monochrome100",
-          borderRadius: "default",
-          filter: `drop-shadow(${theme.shadows?.tooltip})`,
+    <>
+      <div ref={posRef} />
+      {pos ? (
+        <Portal>
+          <Box
+            style={{
+              zIndex: 10,
+              position: "absolute",
+              left: x! + margins.left + pos?.left,
+              top: mxYOffset(y!, placement) + margins.top + pos?.top,
+              pointerEvents: "none",
+              transform: mkTranslation(placement),
+            }}
+          >
+            <Box
+              sx={{
+                width: "fit-content",
+                padding: 3,
+                whiteSpace: "nowrap",
+                pointerEvents: "none",
+                backgroundColor: "monochrome100",
+                borderRadius: "default",
```

Comment:
Is this needed?

Reply:
(This is not code I changed here)

What is "this" in your question ?